mockbee-test/ecom-store/src/components/ProductCard/ProductCard.css

```diff
@@ -0,0 +1,15 @@
+.product-content {
+    margin-top: 5rem;
+    margin-left: 20rem;
+    padding: 2rem 5rem;
+}
+  
+.card-image-container {
+    width: 100%;
+    height: 240px;
```

Comment:
You can use `vh` units for height. or rems to keep it consistent.

Reply:
Sure will do this, generally is used vw, vh or rem but I think I missed replacing pixel here. Thank you for pointing out